src/main/java/com/amazon/opendistroforelasticsearch/knn/plugin/script/KNNScoringSpaceFactory.java

```diff
@@ -35,25 +35,25 @@
  */
 public class KNNScoringSpaceFactory {
     public static KNNScoringSpace create(String spaceType, Object query, MappedFieldType mappedFieldType) {
-        if (KNNConstants.HAMMING_BIT.equalsIgnoreCase(spaceType)) {
+        if (SpaceType.HAMMING_BIT.getValue().equalsIgnoreCase(spaceType)) {
```

Comment:
For another commit:
Shall we create HashMap to store spaceType and instance? We don't have to add if + return whenever we support new space types.

Reply:
Yes, we could in another commit